From cc4402198f6cfd53e0cae540dd0672e7904e7b37 Mon Sep 17 00:00:00 2001
From: ElizandraManuel <elizandramanuel07@gmail.com>
Date: Wed, 27 May 2026 11:43:26 +0100
Subject: [PATCH 1/2] implementacao da classe para persistencia

---
 README.md | 1 +
 1 file changed, 1 insertion(+)

diff --git a/README.md b/README.md
index 85e4878..53e3718 100644
--- a/README.md
+++ b/README.md
@@ -2,3 +2,4 @@
 
 Disciplina: Estrutura de Dados II
 Curso: Engenharia Informática
+
-- 
2.51.0.windows.2

